pcsx2/gui/Panels/GSWindowPanel.cpp

```diff
@@ -140,16 +196,21 @@ void Panels::GSWindowSettingsPanel::ApplyConfigToGui( AppConfig& configToApply,
 
 	if( !(flags & AppConfig::APPLY_FLAG_FROM_PRESET) )	
 	{//Presets don't control these values
-		m_check_CloseGS		->SetValue( conf.CloseOnEsc );
-		m_check_Fullscreen	->SetValue( conf.DefaultToFullscreen );
-		m_check_HideMouse	->SetValue( conf.AlwaysHideMouse );
-		m_check_SizeLock	->SetValue( conf.DisableResizeBorders );
-
-		m_combo_AspectRatio	->SetSelection( (int)conf.AspectRatio );
-		m_text_Zoom			->ChangeValue( conf.Zoom.ToString() );
+		m_check_CloseGS					->SetValue( conf.CloseOnEsc );
+		m_check_Fullscreen				->SetValue( conf.DefaultToFullscreen );
+		m_check_HideMouse				->SetValue( conf.AlwaysHideMouse );
+		m_check_ScalingInterlaced		->SetValue( conf.DisableScalingInterlaceCompensation);
+		m_check_ScalingInterlaced		->Enable( (conf.AspectRatio != AspectRatio_Stretch && conf.ScalingType != ScalingType_Fit) || conf.AspectRatio == AspectRatio_Frame );
+		m_check_SizeLock				->SetValue( conf.DisableResizeBorders );
+
+		m_combo_AspectRatio				->SetSelection( (int)conf.AspectRatio );
```

Comment:
I'd prefer if you could use the generic ``enum_cast()`` function instead of all these manual casts on the enumeration values. This might also help avoid any potential unsigned/signed integer cast conflicts with enumerations.